README.md

```diff
@@ -66,22 +66,31 @@ To begin benchmarking your first embodied agent with NETT, please be aware:
 
 ### Installation
 
-1. **Virtual Environment Setup (Highly Recommended)**
+1. **Virtual Environment Setup**
 
-   Create and activate a virtual environment to avoid dependency conflicts.
+   *(Note: if you are a member of our lab using one of our lab servers, you will need to  initialize Anaconda for your account the first time you use the server for this pipeline: `module load anaconda`)*
```

Comment:
Phrasing should be less lab specific
```suggestion
   *(Note: If anaconda is not already initialized, you may need to do so using: `module load anaconda`)*
```

Reply:
Let's leave this lab specific. We need to leave this information in for our own internal purposes, but we don't know that this will work for other people (it depends on what system they're using and how their servers are configured). Instead of making it sound like this is a general direction for everyone, it's better to make it clear that it's for our internal purposes.